forum/qa-plugin/q2a-googleauthenticator-login/q2a-googleauthenticator-overrides.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+function qa_set_logged_in_user($userId, $handle = '', $remember = false, $source = null)
+{
+    require_once QA_INCLUDE_DIR . 'app/cookies.php';
+    qa_start_session();
+    if (!isset($userId)) {
+        // logout
```

Comment:
Skoro w tym if dzieje się wylogowanie usera, to może zamiast komentarza warto przenieść blok kodu do osobnej metody wylogowującej i ją tu wywołać?

Reply:
To jest override istniejącej funkcji w Q2A - można spróbować coś zmienić, ale trzeba mieć chwilkę na to by to przetestować, czy nie psuje się nic. Swoją drogą, ten redirect który jest tam użyty w innej klasie z formularzem bardzo mi się nie podoba, ale chwilowo nie mam pojęcia jak możnaby to zmienić.